src/pytom_tm/weights.py

```diff
@@ -318,22 +307,21 @@ def create_wedge(
     elif cut_off_radius <= 0:
         raise ValueError("Invalid wedge cutoff: needs to be larger than 0")
 
-    if angles_in_degrees:
-        tilt_angles_rad = [np.deg2rad(w) for w in tilt_angles]
+    if ts_metadata.angles_in_degrees:
+        tilt_angles_rad = [np.deg2rad(w) for w in ts_metadata.tilt_angles]
```

Comment:
```suggestion
        tilt_angles_rad = np.deg2rad(ts_metadata.tilt_angles)
```
I know this is my code, but any reason we could not just do this?

Reply:
nope, added.